src/test/compile-fail/E0089.rs

```diff
@@ -11,7 +11,6 @@
 fn foo<T, U>() {}
 
 fn main() {
-    foo::<f64>();
-//~^ ERROR E0089
-//~| NOTE expected 2 type parameters
+    foo::<f64>(); //~ ERROR expected 2 parameters, found 1 parameter [E0089]
```

Comment:
@GuillaumeGomez it does actually, it's just not repeating "type parameter" throughout the *entire* message:

> error[E0089]: too few **type parameters** provided: expected 2 parameters parameters, found 1 parameter parameter

(emphasis mine)